src/multiversioning.zig

```diff
@@ -578,6 +578,8 @@ pub const Multiversion = struct {
 
     source_buffer: []align(8) u8,
     source_fd: ?posix.fd_t = null,
+    source_offset: ?u64 = null,
+    source_size: ?u64 = null,
```

Comment:
Can pack these fields into `stage.source_read`?

Reply:
Ahh, we need `size` also for `stage.source_open`, then, yeah, perhaps simpler to just keep them as nullable fields